examples/src/index.css

```diff
@@ -81,7 +81,13 @@ blockquote span {
 	outline: none;
 }
 
-/*override semantic-ui grid padding */
-.demo-rich-text-editor {
-	padding: 0px !important; 
-}
\ No newline at end of file
+.heading {
+	font-size: 18px;
+	text-align: center;
+	position: relative;
+	text-transform: uppercase;
+}
+
+.ap-rich-text-editor-toolbar {
+    box-shadow: 0 4px 2px -2px rgba(0,0,0,.2);
+} 
```

Comment:
I think we want to remove this and address in a different issue.

Reply:
agreed

Reply:
Sure!